### PR TITLE
CCXDEV-9284: Clarify SCA entitlement section

### PR DESCRIPTION
**Pull request by xJustin**
Version(s):
enterprise-4.10+

Issue:
https://issues.redhat.com/browse/CCXDEV-9284

Link to docs preview:
https://51013--docspreview.netlify.app/openshift-enterprise/latest/support/remote_health_monitoring/insights-operator-simple-access.html

QE review:
@JoaoFula 
- [x] QE has approved this change.

Additional information:
Small update to clarify the sca section based on customer interactions. 

SME: @tremes

### CODE REVIEW

**Comment by JoaoFula** [comment]
LGTM

**Comment by tremes** [comment]
lgtm too. Thanks

File: modules/insights-operator-configuring-sca.adoc [hunk 1]
```diff
@@ -7,7 +7,7 @@
 [id="insights-operator-configuring-sca_{context}"]
 = Configuring simple content access import interval
 
-You can configure how often the Insights Operator imports the simple content access certificates using the `support` secret in the `openshift-config` namespace. The certificate import normally occurs every 8 hours, but you can shorten this interval if you update your simple content access configuration in Red Hat Subscription Management.
+You can configure how often the Insights Operator imports the simple content access entitlements using the `support` secret in the `openshift-config` namespace. The certificate import normally occurs every 8 hours, but you can shorten this interval if you update your simple content access configuration in Red Hat Subscription Management.
```
Comment (nalhadef) [inline modules/insights-operator-configuring-sca.adoc:10]: Spell out "eight".
This is in the [Red Hat Supplemental Style Guide](https://redhat-documentation.github.io/supplementary-style-guide/#_n).

File: modules/insights-operator-disabling-sca.adoc [hunk 2]
```diff
@@ -7,7 +7,7 @@
 [id="insights-operator-disabling-sca_{context}"]
 = Disabling simple content access import
 
-You can disable the import of simple content access certificates using the `support` secret in the `openshift-config` namespace.
+You can disable the import of simple content access entitlements using the `support` secret in the `openshift-config` namespace.
```
Comment (nalhadef) [inline modules/insights-operator-disabling-sca.adoc:10]: Change "import" to "importing."

File: support/remote_health_monitoring/insights-operator-simple-access.adoc [hunk 3]
```diff
@@ -1,22 +1,27 @@
...
+Insights Operator periodically imports your simple content access entitlements from {cluster-manager-url} and stores them in the `etc-pki-entitlement` secret in the `openshift-config-managed` namespace. Simple content access is a capability in Red Hat subscription tools which simplifies the behavior of the entitlement tooling. This feature makes it easier to consume the content provided by your Red Hat subscriptions without the complexity of configuring subscription tooling.
 
-Insights Operator imports simple content access certificates every 8 hours by default, but can be configured or disabled using the `support` secret in the `openshift-config` namespace.
+Insights Operator imports simple content access entitlements every 8 hours, but can be configured or disabled using the `support` secret in the `openshift-config` namespace.
```
Comment (nalhadef) [inline support/remote_health_monitoring/insights-operator-simple-access.adoc:12]: Spell out "eight."

File: support/remote_health_monitoring/insights-operator-simple-access.adoc [hunk 4]
```diff
@@ -1,22 +1,27 @@
...
+
+[NOTE]
+=====
+Simple content access must be enabled in Red Hat Subscription Management for the import to function.
```
Comment (nalhadef) [inline support/remote_health_monitoring/insights-operator-simple-access.adoc:16]: Change "the import" to "importing."

File: support/remote_health_monitoring/insights-operator-simple-access.adoc [hunk 5]
```diff
@@ -1,22 +1,27 @@
...
+Simple content access must be enabled in Red Hat Subscription Management for the import to function. 
+=====
 
 [role="_additional-resources"]
 .Additional resources
 
 * See link:https://access.redhat.com/documentation/en-us/subscription_central/2021/html-single/getting_started_with_simple_content_access/index#assembly-about-simplecontent[About simple content access] in the Red Hat Subscription Central documentation, for more information about simple content access.
 
-* See xref:../../cicd/builds/running-entitled-builds.adoc[Using Red Hat subscriptions in builds], for more information about using simple content access certificates in {product-title} entitled builds.
+* See xref:../../cicd/builds/running-entitled-builds.adoc[Using Red Hat subscriptions in builds], for more information about using simple content access entitlements in {product-title} builds.
```
Comment (nalhadef) [inline support/remote_health_monitoring/insights-operator-simple-access.adoc:24]: Remove comma after "builds."

**Comment by xJustin** [comment]
Incorporated all suggestions. 

/label merge-review-needed

**Pull request merged**